plugins/modules/rulebook_activation.py

```diff
@@ -24,6 +24,12 @@
       - The name of the rulebook activation.
     type: str
     required: true
+  new_name:
+    description:
+      - Renames an existing rulebook activation.
+      - If set, the rulebook activation will be updated with the new name.
+    type: str
+    version_added: 2.6.0
```

Comment:
We already released 2.6.0, this should be 2.7.0, right?

Reply:
You're, I will adjust it.

Reply:
You're right. I'll adjust it.